keyboards/takashicompany/minidivide_max/info.json

```diff
@@ -0,0 +1,544 @@
+{
+    "manufacturer": "takashicompany",
+    "keyboard_name": "miniDivide MAX",
+    "maintainer": "takashicompany",
+    "build": {
+        "lto": true
+    },
+    "development_board": "promicro",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "oled": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "cols": [
+            "D4",
+            "C6",
+            "D7",
+            "E6",
+            "B4",
+            "B5"
+        ],
+        "rows": [
+            "F4",
+            "F5",
+            "F6",
+            "F7",
+            "B1"
+        ]
+    },
+    "pin_compatible": "promicro",
```

Comment:
```suggestion
```

Reply:
`pin_compatible` defined as part of `development_board` (reference: `data/mappings/defaults.hjson`)